scripts/microbe_stage/organelle_table.as

```diff
@@ -659,6 +676,7 @@ void setupOrganelles(){
     nitrogenFixationProtien.chanceToCreate = 0;
     nitrogenFixationProtien.prokaryoteChance = 1;
     nitrogenFixationProtien.mpCost = 15;
+    nitrogenFixationProtien.needsNucleus = false;
```

Comment:
One suggestion i have would be to remove this parameter entirely and just check if prokaryote chance is greater then 0. Would accomplish the same thing with less code.